readthedocs/subscriptions/products.py

```diff
@@ -87,10 +110,8 @@ def get_feature(obj, feature_type) -> RTDProductFeature:
     :param obj: An organization or project instance.
     :param type: The type of the feature (readthedocs.subscriptions.constants.TYPE_*).
     """
-    # Hit the DB only if subscriptions are enabled.
-    if settings.RTD_PRODUCTS:
-        from djstripe import models as djstripe
-
+    # Hit the DB only if subscriptions and organizations are enabled.
+    if settings.RTD_PRODUCTS and settings.RTD_ALLOW_ORGANIZATIONS:
```

Comment:
Seems like it would be clearer to invert this logic and just return when it isn't allowed?